lms/static/scripts/frontend_apps/api-types.ts

```diff
@@ -208,8 +208,36 @@ export type AssignmentWithMetrics = Assignment & {
 };
 
 /**
- * Response for `/api/dashboard/courses/{course_id}/assignments/stats` call.
+ * Response for `/api/dashboard/courses/{course_id}/assignments/metrics` call.
  */
-export type AssignmentsResponse = {
+export type AssignmentsMetricsResponse = {
   assignments: AssignmentWithMetrics[];
 };
+
+export type Pagination = {
+  next: string | null;
+};
+
+/**
+ * Response for `/api/dashboard/courses call.
+ */
+export type CoursesResponse = {
+  courses: Course[];
+  pagination: Pagination;
+};
+
+/**
+ * Response for `/api/dashboard/assignments call.
+ */
+export type AssignmentsResponse = {
+  assignments: Assignment[];
+  pagination: Pagination;
+};
+
+/**
+ * Response for `/api/dashboard/courses call.
```

Comment:
```suggestion
 * Response for `/api/dashboard/students` call.
```

I assume this is what you meant?

Reply:
Oops! Yes 😅 

I'm not completely sure how useful are these comments though. They quickly get outdated and can do more harm than good.